service-registry/src/main/java/io/servicecomb/serviceregistry/task/MicroserviceRegisterTask.java

```diff
@@ -113,8 +116,8 @@ private boolean checkSchemaIdSet() {
     schemaIdSetMatch = existSchemas.equals(localSchemas);
 
     if (!schemaIdSetMatch) {
-      LOGGER.error(
-          "SchemaIds is different between local and service center. Please change microservice version. "
+      LOGGER.warn(
```

Comment:
product mode, it's a error, why change to warn?
and must stop boot.

Reply:
done

Reply:
???
product mode , if check failed, must stop boot?

i can not find the logic.